README.rst

```diff
@@ -503,3 +496,12 @@ It will always use ``EUR`` for all ``Money`` instances when ``obj.money`` is cal
 
     class Expenses(models.Model):
         amount = MoneyField(max_digits=10, decimal_places=2, money_descriptor_class=MyMoneyDescriptor)
+
+
+Background
+----------
+
+This project is a ork of the Django support that was in
```

Comment:
Typo “ork” -> “fork”

Reply:
Haha, thanks for noticing... I really wonder what happened as I was using Nano and CTRL+K then CTRL+U :)

Reply:
:))